spec.bs

```diff
@@ -1295,12 +1317,22 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       with |biddingSignalsUrl| and true.
     1. If |dataVersion| is not null:
       1. [=map/Set=] |browserSignals|["{{BiddingBrowserSignals/dataVersion}}"] to |dataVersion|.
+    1. Let |fetchSignalDuration| be the [=duration from=] |fetchSignalStartTime| to
+      [=current wall time=], in milliseconds.
+    1. If |perBuyerCumulativeTimeout| is less than |fetchSignalDuration|, then [=iteration/break=];

```

Comment:
So I think this doesn't quite work since you only break the inner loop and don't actually decrement perBuyerCumulativeTimeout, so even though it expired it will still try to do stuff?

Reply:
I think it's not quite enough since it will still try to fetch things?

Reply:
Oh, I may have misread the loop level here...